app/presenters/apartment_presenter.rb

```diff
@@ -0,0 +1,34 @@
+class ApartmentPresenter
```

Comment:
Presenter patterns can be great, and this may indeed be a good fit here, but I'm interested whether view helpers may sense for these methods (they may not, for a variety of reasons, I'm just flagging this out of curiosity).